docs/getting_started.rst

```diff
@@ -43,6 +71,21 @@ From here, we can also convert our ``Pauli`` object back to a dense numpy array
 Pauli Strings
 ------------------------
 
+Pauli strings, also known as Pauli words, are tensor-product combinations of Pauli matrices. For example, the following is a valid Pauli string:
+
+.. math::
+
+    \mathcal{\hat{P}} = \sigma_x \otimes \sigma_y \otimes \sigma_z
+
+where :math:`\otimes` denotes the tensor product, and we can more simply denote by
+
+.. math::
+
+    \mathcal{\hat{P}} = XYZ
+
+Other valid Pauli strings include ``III``, ``IXYZ``, ``IZYX``, etc. In general, a Pauli string of length ``N`` is a tensor product of ``N``
+Pauli matrices. A ``N``-length Pauli String in dense form is a :math:`2^N \times 2^N` matrix, so ``XYZ`` is a :math:`8 \times 8` matrix.
```

Comment:
I don't think you need to mention dense form, the idea of dense versus sparse representation is more of how we write it in memory and not an actual mathematical term.